circuit-breaker/README.md

```diff
@@ -52,40 +52,105 @@ In terms of code, the end user application is:
 
```

Comment:
In the `Real world example` let's describe the application we have implemented here with the monitoring service calling the quick and delayed remote services

Reply:
`ServiceDiagram.PNG` seems to be missing the quick remote service

Reply:
Updated `Real world example` description
`ServiceDiagram.png` updated to include quick service